server/cluster_test.go

```diff
@@ -545,4 +545,35 @@ func TestClusterResize_RemoveNode(t *testing.T) {
 			t.Fatalf("expected Body '%s' but got '%s'", expBody, strings.TrimSpace(resp.Body))
 		}
 	})
+
+	t.Run("ErrorRemoveWithoutReplicas", func(t *testing.T) {
+		client0 := m0.Client()
+
+		// Create indexes and frames on one node.
+		if err := client0.CreateIndex(context.Background(), "i", pilosa.IndexOptions{}); err != nil && err != pilosa.ErrIndexExists {
+			t.Fatal(err)
+		} else if err := client0.CreateFrame(context.Background(), "i", "f", pilosa.FrameOptions{}); err != nil {
+			t.Fatal(err)
+		}
+
+		setBits := ""
+		for i := 0; i < 20; i++ {
```

Comment:
let's add a comment here noting that this is an attempt to ensure there is data on both nodes, but it's not guaranteed.